### ISSUE TITLE
(Self-Voicing) The Close button on the dialogs do not self-voice

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#261]
@jessegreenberg, this is likely not a priority for interviews. I noticed that the X (close) button for the  Gestures dialog and the Preferences dialog do not self-voice, "Close".

Since these dialogs are temporary in nature, we might want to just inform @liquatics about that so she can watch for this if participants need to close these dialogs. 

The Gestures dialog has the "Continue to Simulation" button, but the Preferences dialog, only has the Close button.

**Comment by terracoda** [comment]
@jessegreenberg, if the Close button lives in the joist repo, I think it is safe to assume we can leave the button as silent for now, and @liquatics will watch for this in interviews.

**Comment by terracoda** [comment]
Un-assigning you both. You are both tagged and will be made of the issue.

**Comment by jessegreenberg** [comment]
Thanks @terracoda I will take a look - this button is in joist but I had thought that I added something that would read the label anway. It isn't clear to me why it is working in one dialog but not others.

**Comment by jessegreenberg** [comment]
I just tested on master and I hear self-voicing for the "Web Speech!" dialog, the "Gestures" and "Preferences" dialog. @terracoda can you let me know what version of the sim you were using and on what platform/browser?

**Comment by terracoda** [comment]
@jessegreenberg, the general self-voicing of all content is working great. 
I was listening in Test Flight, and I hear nothing if I put focus on the close buttons.
I just listened to 1.1.0-dev.30 on phettest and in both cases I don't hear "Close" when I put focus on the X in any dialogs.

Maybe it is an iOS problem?

**Comment by terracoda** [comment]
I have iOS 13.5.1 on the iPhone8.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, yes it looks like it is something platform specific. I am also not hearing "Close" on those dialogs on an iPhone.

**Comment by jessegreenberg** [comment]
Hmm, I cannot get this to happen in iOS Safari, but it is happening in the app. So something about the TestFlight app is causing this to happen.

**Comment by terracoda** [comment]
@jessegreenberg, maybe the way forward is to just make @liquatics aware of this issue. These dialogs are implemented to support use of the sims with haptics in upcoming interviews. The dialogs in their current form will likely not be part of a published sim.

**Comment by jessegreenberg** [comment]
Yea sounds good - it doesn't seem like this is critical. But even more strange is that it is not happening off of master, but is happening in the latest version published for test flight. There is a chance this has just been fixed in master, but I don't know what has changed to to cause that.

**Comment by terracoda** [comment]
I think this issue might be stale as the Preferences Menu has seen many updates.

**Comment by terracoda** [comment]
I will close and open new issues re Preferences Menu in the joist repo.
